projects/social/social.py

```diff
@@ -45,8 +60,24 @@ def populate_graph(self, num_users, avg_friendships):
         # !!!! IMPLEMENT ME
 
         # Add users
+        for i in range(num_users):
+            self.add_user(f'user{i}')
+
+        print(self.users)
+        
 
         # Create friendships
+        possible_friendships = []
+        for user1 in self.users:
+            for user2 in self.users:
+                if user1 < user2 and [user1, user2] not in possible_friendships:
```

Comment:
Nicely done! Is `[user1, user2]` an array checking for a tuple in possible_friendships?